### ISSUE TITLE
Missing example with "use" statemenets and lines

### ISSUE THREAD

**Issue by GEp1c** [nette/coding-standard#26]
For ex. likes this https://github.com/nette/tracy/blob/master/src/Tracy/Debugger.php#L8-L17

**Comment by GEp1c** [comment]
But if I look at the files in src folder, it's not clear too, sometimes there are 2 likes between "class" and "use", sometimes only 1. Does it really matter or not?

**Comment by TomasVotruba** [comment]
There is no rule which would enforce this yet.

Personally **I'd prefer exactly 1 space everything**, since it's more open to English-only PHP world and code style readability is rather responsbility of IDE: fontsize, lineheight, colors and display resolution.

That could be done via: [`PhpCsFixer\Fixer\Import\SingleLineAfterImportsFixer`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.10/src/Fixer/Import/SingleLineAfterImportsFixer.php)

Here is discussion if you want to express your view: https://github.com/nette/coding-standard/pull/24

**Comment by dg** [comment]
> it's more open to English-only PHP world

Can you explain it?

**Comment by TomasVotruba** [comment]
@dg Sure. Top of European and American open-source projects on Github uses PSR-2 with single blank spacing for defaults ([mentioned here](http://www.php-fig.org/psr/psr-2/#1-overview) + [used in examples](http://www.php-fig.org/psr/psr-2/#11-example)). And not only the code, but same applies for blogs, tutorials and so on.

I only see such custom coding standard in Czech projects, which makes it harder to read for abroad-people where 95 % of code you see in everyday life respects PSR-2. This is typical for *[blind leading the blind effect](https://en.wikipedia.org/wiki/The_blind_leading_the_blind)* and **I didn't realize this myself until I started to contribute to projects abroad**. 

First it was pissing me off: "why they don't use common standard and make their own that nobody uses?". But then I realized I was blindfoleded by Czech-only code.

I feel I should remind that this **is not about standards** that other people made and you're forced to use them, just because they were first to talk about such *standard*.

But rather about **improving understanding the code by world** and vise versa. Which logically leads to better addoption, contributing and satisfaction with the software. It would allow to spread this code a little bit easily to other projects.

It would also prevent [questions like this](https://github.com/nette/coding-standard/issues/26) and people could contribute with much more confidence. Seeing other non-mainstream PHP frameworks used in the worlds, I believe Nette has potential to expand much more while being friendly to aborad contributors.

*Note: this all is impossible to see without contributing to abroad projetcs.*
